environ.bzl

```diff
@@ -0,0 +1,70 @@
+# -*- mode: python -*-
```

Comment:
Move to `//tools` for now, though I may move/replace it once I have support (almost complete) for downloading both Linux and macOS binary package automatically.

Reply:
Done.  (i took you literally, and put it in `//tools`.  not sure if you meant `//tools/rt`.